Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -806,6 +806,17 @@
     return std::nullopt;
 }
 
+std::optional<RetainPtr<PDFAnnotation>> UnifiedPDFPlugin::annotationForRootViewPoint(const WebCore::IntPoint& point) const
+{
+    auto pointInDocumentSpace = convertFromPluginToDocument(convertFromRootViewToPlugin(point));
+    auto nearestPageIndex = nearestPageIndexForDocumentPoint(pointInDocumentSpace);
+    if (!nearestPageIndex)
+        return std::nullopt;
+
+    auto page = m_documentLayout.pageAtIndex(nearestPageIndex.value());
+    return [page annotationAtPoint: convertFromDocumentToPage(pointInDocumentSpace, nearestPageIndex.value())];
```

Comment:
No space in objC calls

```suggestion
    return [page annotationAtPoint:convertFromDocumentToPage(pointInDocumentSpace, nearestPageIndex.value())];
```

Reply:
Fixed! I wish the styler checker would catch these :(

Reply:
+1, could you file a bugzilla?